aii-core/src/main/eg/aii-shellfe.conf

```diff
@@ -15,3 +15,22 @@
 # The default is profile_ for backward compatibility
 # Comment this out when not prefix is needed
 profile_prefix = profile_
+
+# Directory on TFTP server where PXELINUX configuration files are.
+# Must be an absolute path or none to disable PXELINUX configuration.
+# nbpdir = /osinstall/nbp/pxelinux.cfg
+
+# Directory on TFTP server where Grub2 PXE configuration files are.
+# Must be an absolute path or none to disable Grub2 PXE configuration.
+# Default value is based on 'pxelinux' option if defined with last
```

Comment:
if i understand correctly, this is better phrased as `based on PXELINUX option 'nbpdir'`; and for readability, there should be a `,` after `defined`?
i'll read the code later, maybe that will make me understand it.